zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -434,6 +434,7 @@ def make_tasks_with_checkerboarding(  # pylint: disable=too-many-locals, too-man
             backend chunk size to half of the processing chunk size. Furthermore, skip caching
             if the temporary destination happens to be local.
             """
+            assert dst is not None  # (impossible to get a temp dst without dst and intermediaries)
```

Comment:
This doesn't need a comment - if we see `assert foo is not None` in the code, we know that it's to get around typing for an optional kwarg. (The context is that we want to keep the number of comments in the code to a minimum.)